### DISCUSSION TITLE
Audio Glitches After 20210528 Update

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#343] [Fixed Issues]
@rern
I am getting audio interference when the UI changes, and I think also when the USB storage is addressed, since today's update. No problems with a 20210502 image.

Using Pi3B rev1.2 with USB DAC, USB storage, and USB wifi dongle.

**Comment by R101G** [comment]
Fixed by 20210530 update.

**Reply by R101G** [reply]
Problem is still there with 64bit version. I suspect the USB wifi dongle has too much priority.

**Comment by R101G** [comment]
This problem is not present when using the 32bit image, so my easiest solution is to abandon the 64bit version.

**Comment by R101G** [comment]
@rern  Sorry for the confusing reports. Using the 20210502 update, the glitches occur when the local browser is disabled, followed by a reboot. Enabling the local browser clears the problem, with no reboot needed.

**Reply by rAndyTBKG** [reply]
Replying to a somewhat old thread... I frequently get glitches with my old PiB and the 20231022 release when playing CD quality FLACs. Especially when navigating the web GUI (via Safari on Macbook and wifi) the audio stutters, but also intermittently and less pronounced when not using the GUI.  

The "local browser" setting seems to have been removed since last message. Is there some other setting I can use to improve the situation? Or am I hopelessly lost with my old RPiB?

rAudio 20231022
6.1.34-1-rpi-legacy-ARCH armv6l
Raspberry PiB Rev 2
BCM2835 • 455 MiB
ARM1176 @ 700 MHz

**Reply by R101G** [reply]
@rAndyTBKG 
I think you could be right in thinking that rAudio, or its OS, has now outgrown your old Pi. You might have to make do with an old image. Try putting a future date in the /srv/http/data/addons/r1 file to prevent updating.

**Reply by R101G** [reply]
Something else worth trying if the Pi is using wifi is to check if the problem is still there when you use a wired connection.

**Reply by rAndyTBKG** [reply]
Thanks!.Unfortunatelly, my Pi is on wire to my wifi-router, so no chances there. I saw a suggestion somewhere to rename the “local browser”-Setting. Is it still around under different name?  Could it be tweaked to improve performance?

**Reply by R101G** [reply]
I think the local browser setting was removed from the low-power Pi images because they could no longer cope with it. You could try a custom build, but I have never succeeded in doing that myself.

**Reply by rern** [reply]
@rAndyTBKG,

Please try the [latest image](https://github.com/rern/rAudio/releases/tag/i20240108) which plays FLACs fine on my Zero.
